src/ape_test/runners.py

```diff
@@ -67,10 +58,10 @@ def pytest_sessionstart(self):
         so related assertions cannot be rewritten". The warning is not relevant
         for end users who are performing tests with ape.
         """
-        reporter = self.config.pluginmanager.get_plugin("terminalreporter")
```

Comment:
**HEY**  this `config` is a `PytestConfig`, not a `ape.managers.ConfigManager`.
I feel like that will get confusing.

Reply:
Now you see why I suggested name change to `*_manager` for everything

Reply:
Yeah, I did get very confused about that. I now see that config wasn't the config_manager. LOL